AllReadyApp/Web-App/AllReady/Hangfire/Jobs/SendRequestStatusToGetASmokeAlarm.cs

```diff
@@ -15,14 +15,17 @@ public class SendRequestStatusToGetASmokeAlarm : ISendRequestStatusToGetASmokeAl
         public SendRequestStatusToGetASmokeAlarm(IOptions<GetASmokeAlarmApiSettings> getASmokeAlarmApiSettings, IHttpClient httpClient)
         {
             _getASmokeAlarmApiSettings = getASmokeAlarmApiSettings.Value;
+            _httpClient = httpClient;
         }
 
         public void Send(string serial, string status, bool acceptance)
         {
-            var updateRequestMessage = new { acceptance, status };
-
+            var str = JsonConvert.SerializeObject(new { acceptance, status });

```

Comment:
@chinwobble, thanks for the explanation. In the end, it's probably better to have it there as it could potentially catch bad JSON before i goes out the door to the web call, and like you said, HttpClient was already doing it. Thanks!